src/twind/plugins.ts

```diff
@@ -662,10 +658,7 @@ export const corePlugins: Plugins = {
       case 'gradient':
         if (params[1] === 'to' && (_ = expandEdges(params[2]))) {
           return {
-            backgroundImage: `linear-gradient(to ${join(
-              _,
-              ' ',
-            )},var(--tw-gradient-stops,var(--tw-gradient-from,transparent),var(--tw-gradient-to,transparent)))`,
+            backgroundImage: `linear-gradient(to ${join(_, ' ')},var(--tw-gradient-stops)`,
```

Comment:
Hey, I know this merged but I think this causes a bug: 

There should be two closing braces towards the end:

```backgroundImage: `linear-gradient(to ${join(_, ' ')},var(--tw-gradient-stops))```

Reply:
Great catch! I‘ll update that and add an ui test.

Reply:
fixed  in 0f3264c